app-editors/featherpad/featherpad-0.18.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg qmake-utils
+
+MY_P="${P/featherpad/FeatherPad}"
```

Comment:
Since this variable is only used once in this ebuild, you can just plaintext it in the part where it's used. We usually define these kind of variables when they get used multiple times in an ebuild.

Reply:
But then I have to maintain the version number in the ebuild file (since the unpacked tar.gz file contains the version number). I think I'll keep it. Or is there a better way to resolve this?

Reply:
`S="${WORKDIR}/FeatherPad-${PV}"`